src/cmap/commands.ts

```diff
@@ -826,7 +827,7 @@ export class BinMsg {
     while (this.index < this.data.length) {
       const payloadType = this.data.readUInt8(this.index++);
       if (payloadType === 1) {
-        console.error('TYPE 1');
+        throw new MongoError('Type 1 payload'); // ???
```

Comment:
Nice find, good call restricting the console functions in the linter. Can we leave a FIXME or TODO comment here instead of question marks? Or maybe it's even worth filing a JIRA ticket to investigate this?

Reply:
Sync-ed in the channel on this, left a comment describing this is an used part of the protocol, and made the error message a bit more clear.